### ISSUE TITLE
Reader Conversations: Add Follow Conversation button

### ISSUE THREAD

**Issue by jancavan** [Automattic/wp-calypso#17668]
We've now added the Follow(ing) Conversation icons to the Gridicons set in: https://github.com/Automattic/gridicons/pull/246

Next step are:

1. Add a "Follow(ing) Conversation" button in the popover:

![screenshot 2017-08-30 19 28 33](https://user-images.githubusercontent.com/4924246/29903843-e7bd4e26-8dba-11e7-8eda-38c5a0b6facf.png)

![screenshot 2017-08-30 19 40 29](https://user-images.githubusercontent.com/4924246/29903893-18ea8680-8dbb-11e7-86f0-97962ead9145.png)

We should also update "Follow" to "Follow Site" to avoid confusion of what you're following.

2. Add a "Follow(ing) Conversation" button opposite the Comments header in fullpost:

![screenshot 2017-08-30 19 34 32](https://user-images.githubusercontent.com/4924246/29903930-3c68c590-8dbb-11e7-8d3f-dacab6202960.png)
![screenshot 2017-08-30 19 34 36](https://user-images.githubusercontent.com/4924246/29903929-3c4819f8-8dbb-11e7-9d6c-26fc8b537bf8.png)

**Comment by fraying** [comment]
@jancavan This is perfect. Thank you!

Only one note: In the mockups, when the link is green and says "Following Conversation," you've got the bubble/plus icon showing. I think you meant to use the bubble/check version, yes?

**Comment by fraying** [comment]
One question: In the Fullpost version, after the "Follow Conversation" link is clicked, should we prompt the user to go to Conversations somehow? 

We don't do this when a user follows a site in Reader, but all they have to do is go back to the top of Reader to see what they followed. For a Conversation, they have to know to go to Conversations.

I wonder if, after clicking "Follow Conversation," we should prompt the user with a link to visit Conversations. Maybe in a bubble, similar to the way we do it on the frontend?

Just noting this as a thing to think about. cc @jancavan

**Comment by jancavan** [comment]
> I think you meant to use the bubble/check version, yes?

Yes, good catch. Updated!

> I wonder if, after clicking "Follow Conversation," we should prompt the user with a link to visit Conversations. Maybe in a bubble, similar to the way we do it on the frontend?

Good idea. So something like this when they're in fullpost:
![screenshot 2017-09-04 15 47 52](https://user-images.githubusercontent.com/4924246/30039609-71de89d2-9188-11e7-9c97-edae212116d2.png)

When they follow from the popover, we show a bubble that animates into place. We won't be able to do this on mobile though so perhaps we can just use the global notice component for that.

![screenshot 2017-09-04 16 08 49](https://user-images.githubusercontent.com/4924246/30039843-79b8a5ae-918b-11e7-80d7-538c71d42a00.jpg)

And I think maybe after 3 convos or so (?), we no longer show the popover and we can switch to a global notice component.

**Comment by fraying** [comment]
@jancavan Love it. 👍

**Comment by bluefuton** [comment]
One last thing to decide here: how we allow users to mute conversations shown in Conversations that they aren't explicitly following. 

Do we just show the 'Following Conversation' button for every post as if they were following it?

**Comment by jancavan** [comment]
> Do we just show the 'Following Conversation' button for every post as if they were following it?

@bluefuton Yes, the button in the ellipsis should be green if they are following it, regardless if they had explicitly clicked on the button or if they're following it because they had Liked or Commented on that post.

If they want to "mute" it, they should just unfollow the conversation.

**Comment by bluefuton** [comment]
> Yes, the button in the ellipsis should be green if they are following it, regardless if they had explicitly clicked on the button or if they're following it because they had Liked or Commented on that post.

Makes sense. We can do that quite easily in Conversations itself, but outside of Convos, we don't actually know what the implicitly followed conversations are, so the button status might be wrong :/

**Comment by samouri** [comment]
@bluefuton: can we close this in lieue of a more specific issue regarding correct follow status?
